operator/config/olm-catalog/ibm-spectrum-scale-csi-operator/2.4.0/csiscaleoperators.csi.ibm.com.crd.yaml

```diff
@@ -1,266 +0,0 @@
----
```

Comment:
we need to retain csv file for 2.4.0 release

Reply:
Done